src/Platforms/AbstractPlatform.php

```diff
@@ -2477,6 +2477,11 @@ public function getCreateIndexSQL(Index $index, $table)
         $name    = $index->getQuotedName($this);
         $columns = $index->getColumns();
 
+        if ($this instanceof SqlitePlatform && strpos($table, '.') !== false) {
```

Comment:
This is the only instanceof check against a specific platform in the whole file. :nauseated_face: 

Alternatively, I could copy the whole method to the platform file, or introduce a new extension point. Let me know what would be best please.

Reply:
I guess, copying this down to the SQLite platform is the way to go for a bugfix. We might consider refactoring this on a higher branch though.